client/container/EventDetail.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import Axios from 'axios';
+import Highlight from '../components/Highlights/Highlights';
+import EventImage from '../components/EventImage/EventImage';
+
+const EventDetail = ({ location }) => {
+  const [event, setEvent] = useState(null);
+  const [guarantee, setguarantee] = useState(null);
+  // const [description, setDescription] = useState('');
+  // const [highlights, setHighLights] = useState([]);
+  // const [location, setLocation] = useState([]);
+
+  useEffect(() => {
+    fetchFromDB('event/43', setEvent);
+  }, []);
+
+  useEffect(() => {
+    fetchFromDB('guarantee', setguarantee);
+  }, []);
+
+  const fetchFromDB = (path, cb) => {
+    Axios.get(`/api/${path}`)
+      .then(({ data }) => {
+        console.log('TCL: fetchFromDB -> data', data);
+        cb(data);
+      })
+      .catch((err) => {
+        console.log('TCL: fetchFromDB -> err', err);
+      });
+  };
+
+  return (
+    <>
```

Comment:
Should the empty brackets `<>` be `<div>` instead?

Reply:
Ah, no.  That's valid `JSX.`  

the `<>` and `</>` is sugar for `<Fragment />`  or `<React.Fragment />`.

[Here is a reference for you ](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html)

Any other thoughts?

Reply:
No, that was the only thing that caught my eyes.